cmd/node/main.go

```diff
@@ -819,6 +828,11 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 	err = ioutil.WriteFile(statsFile, []byte(sessionInfoFileOutput), os.ModePerm)
 	log.LogIfError(err)
 
+	computedRatingsData := filepath.Join(statsFolder, "ratings.info")
+	computedRatingsDataStr := createStringFromRatingsData(ratingsData)
+	err = ioutil.WriteFile(computedRatingsData, []byte(computedRatingsDataStr), os.ModePerm)
```

Comment:
I actually like it that we persist the computed rating parameters to a file
The rating is important and this way it is easy to check afterwards.